pkg/executor/client/job.go

```diff
@@ -153,7 +153,7 @@ func (c *JobExecutor) Logs(ctx context.Context, id string) (out chan output.Outp
 
 		for l := range logs {
 			entry, err := output.GetLogEntry(l)
-			if err != nil {
+			if err != nil && entry.Type_ != output.TypeUnknown {
```

Comment:
will we have not nil entry there always?

Reply:
GetLogEntry is returning a testkube.ExecutorOutput type not a pointer.